rules/config

```diff
@@ -117,7 +117,7 @@ DEFAULT_VS_PREPARE_MEM = yes
 
 
 # ENABLE_ICCPD - build docker-iccpd for mclag support
-ENABLE_ICCPD = n
+ENABLE_ICCPD = y
```

Comment:
can you keep this disabled? I think we need a way to build them by default, however, keep the docker in disable state until user enable it explicitly at the runtime. before we have that mechanism, all the option features should in the disabled state, including sflow, nat.

Reply:
Yes, but in PR it is enabled to build and test, or else all checks will be meaningless.